### PR TITLE
Fixes for rmarkdown syntax errors in roxygen documentation code

### PR DESCRIPTION
**Pull request by remlapmot**
When reading your helpfiles I noticed that you had used a slightly incorrect syntax for your links. So I have fixed them with this PR. 

cheers
  Tom

### CODE REVIEW

File: R/BiocManager-pkg.R [hunk 1]
```diff
@@ -13,25 +13,25 @@ NULL
 #' Main functions are as follows; additional help is available for
 #' each function, e.g., `?BiocManager::version`.
 #'
-#' - `BiocManager::install()`: Install or update packages from
+#' - [`BiocManager::install()`]: Install or update packages from
```
Comment (mtmorgan) [inline R/BiocManager-pkg.R:16]: These are meant to be formatted as code giving the fully resolved command `BiocManager::install()`. The update formats this as `install()`. The reason for using the fully resolved command is that represents 'best practice' -- no ambiguity about which package `install()` comes from. 

Also, it seemed to me that it was more helpful to have the code-formatted fully resolved command, than to aim for the fully formatted command AND link, which would render in a confusing way.

This comment applies throughout.

File: R/BiocManager-pkg.R [hunk 2]
```diff
@@ -40,10 +40,10 @@ NULL
 #'
 #' - `"repos"`, `"BiocManager.check_repositories"`,
 #'   `"BiocManager.snapshot"`: URLs of additional repositories for use
-#'   by `BiocManger::install()`. See `?repositories`.
+#'   by [`BiocManager::install()`]. See `?repositories`.
 #'
 #' - `"pkgType"`: The default type of packages to be downloaded and
-#'   installed; see `?install.packages`.
+#'   installed; see [`install.packages`].
```
Comment (mtmorgan) [inline R/BiocManager-pkg.R:46]: The current formatting indicates how one is to get help on `install.packages` and so is informative across help systems, including terminal-based help systems that don't render links.

File: R/install.R [hunk 3]
```diff
@@ -274,20 +274,20 @@
 #' Installation of _Bioconductor_ and CRAN packages use R's standard
 #' functions for library management -- `install.packages()`,
 #' `available.packages()`, `update.packages()`. Installation of GitHub
-#' packages uses the `remotes::install_github()`.
+#' packages uses the [`remotes::install_github()`].
 #'
 #' When installing CRAN or _Bioconductor_ packages, typical arguments
-#' include: `lib.loc`, passed to `\link{old.packages}()` and used to
+#' include: `lib.loc`, passed to [`old.packages()`] and used to
```
Comment (mtmorgan) [inline R/install.R:280]: thanks, `` `link(...)()` `` changes look like the right thing to do...

File: R/install.R [hunk 4]
```diff
@@ -334,19 +334,19 @@
 #' @return `BiocManager::install()` returns the `pkgs` argument, invisibly.
 #' @seealso
 #'
-#' `BiocManager::\link{repositories}()` returns the _Bioconductor_ and
+#' [`BiocManager::repositories()`] returns the _Bioconductor_ and
```
Comment (mtmorgan) [inline R/install.R:337]: I think this and similar, below, would be better as `` `BiocManager::repositories()` ``, without the link, following the logic in the comment above. Alternatively, I think they should be formatted with the fully resolved function `BiocManager::repositories()`, and with 'repositories' linked to the help page -- old-school `\code{BiocManager::\link{repositories}()}` I think.

File: R/valid.R [hunk 5]
```diff
@@ -56,47 +56,47 @@
...
 #'     and _Bioconductor_ currently in use.
 #'
-#'     Packages are reported as 'out-of-date' if a more recent version
-#'     is available at the repositories specified by
-#'     `BiocManager::repositories()`.  Usually, `BiocManager::install()` is
-#'     sufficient to update packages to their most recent version.
+#' Packages are reported as 'out-of-date' if a more recent version
+#' is available at the repositories specified by
+#' [`BiocManager::repositories()`].  Usually, [`BiocManager::install()`] is
+#' sufficient to update packages to their most recent version.
```
Comment (mtmorgan) [inline R/valid.R:65]: Please maintain indentation.

**Review by mtmorgan — CHANGES_REQUESTED** [review]
Thanks for the changes. I agree with some but have reservations about others, as indicated in the comments in the code.

**Comment by LiNk-NY** [comment]
Closed for inactivity.

**Pull request closed without merging**